src/api/entities/__tests__/Account.ts

```diff
@@ -23,7 +23,7 @@ describe('Account class', () => {
     address = 'someAddress';
     key = 'someKey';
 
-    sinon.stub(utilsConversionModule, 'addressToKey').withArgs(address).returns(key);
+    sinon.stub(utilsConversionModule, 'addressToKey').returns(key);
```

Comment:
Shouldn't it be `.withArgs(account, context)`?

Reply:
no need to check attrs in this one, the return value will be always the same